package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "messari-subgraph-cli": "^0.1.52",
```

Comment:
Asking for a friend, why would we want this in the package.json?

Reply:
Don't need it!